port/zos390/omrvmem_support_above_bar.s

```diff
@@ -1,5 +1,5 @@
 ***********************************************************************
```

Comment:
Since omrvmem_support_above_bar.s is replaced by using metalc to compile omriarv64.mc, this file should be deleted.

Reply:
Done in 46472651a0b1614dab255a60946dcbcc492824ef